client/src/components/overview/GalleryView.jsx

```diff
@@ -33,6 +33,20 @@ const StyledOverviewGrid = styled.div`
 
 function Overview(props) {
   const [count, setCount] = useState(0);
+  useEffect(() => {
+    axios({
+      method: 'get',
+      url: 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/products/37314/styles',
+      headers: {
+        Authorization: API_KEY
+      }
+    })
+      .then((data) => {
+        console.log(data.data.results);
```

Comment:
console.log?

Reply:
I believe I got rid of the console logs later on. I double checked with my current files and there are none.